pkg/querier/querier.go

```diff
@@ -36,6 +36,12 @@ type Config struct {
 	// step if not specified.
 	DefaultEvaluationInterval time.Duration
 
+	// Directory for ActiveQueryTracker. If empty, ActiveQueryTracker will be disabled and MaxConcurrent will not be applied (!).
+	// ActiveQueryTracker logs queries that were active during the last crash, but logs them on the next startup.
+	// However, we need to use active query tracker, otherwise we cannot limit Max Concurrent queries in the PromQL
+	// engine.
+	ActiveQueryTrackerDir string `yaml:"active_query_tracking_dir"`
```

Comment:
`active_query_tracking_dir` > `active_query_tracker_dir` to be consistent with the CLI flag.